### PR TITLE
Modified routes, footer, added new feature LikesList

### PR DESCRIPTION
**Pull request by Danulo004**

### CODE REVIEW

File: client/src/components/LikesList.vue [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  },
+  methods: {
+  },
+  mounted () {
+    likes.sort((a, b) => {
+      return b.date - a.date
+    })
+  },
+  filters: {
+    howMatchTimeAgo: function (date) {
```
Comment (de5peratelighter) [inline client/src/components/LikesList.vue]: This is "developer's nightmare" - please don't reinvent the wheel.
Moment.js has "moment(new Date(timeStamp)).fromNow()" functionality.

On real projects you will be forced to do translations of such things yourself unless library has it covered (and moment has all math and translations in cases of importing and properly switching them)

File: client/src/layouts/Footer.vue [hunk 2]
```diff
@@ -2,26 +2,26 @@
...
     <q-footer>
       <q-tabs>
-        <q-tab name="feed">
-          <router-link to="/Feed">
-            <q-icon name="videocam" size="1.5em"/>
-          </router-link>
-        </q-tab>
-        <q-tab name="add">
-          <router-link to="/AddPost">
+        <router-link to="/Feed">
```
Comment (de5peratelighter) [inline client/src/layouts/Footer.vue]: For second demo this is a good idea to display all lists via v-for (will be based on array of objects with next route and icon)
Moreover I believe that **q-tabs** requires **q-tab** to be a direct child component, but more curiously there is a **q-route-tab** component made specifically for this case

File: client/src/views/Feed/Feed.vue [hunk 3]
```diff
@@ -25,13 +25,13 @@
...
-            >
+            <div class="heart">
               <i class="far fa-heart fa-lg"
-                :class="{'fas': post.hasBeenLiked}">
+                :class="{'fas': post.hasBeenLiked}"
+                 @click="updateLikes({ key: key, updates: ({ hasBeenLiked : post.hasBeenLiked, likes: post.likes })  })"
+              >
               </i>
-              <p class="likes">{{post.likes}}</p>
+              <p class="likes" @click="likesInfo.show=true; play">{{post.likes}}</p>
```
Comment (de5peratelighter) [inline client/src/views/Feed/Feed.vue:27]: I hope user won't be forced to click "like" to play the video;)

**Review by de5peratelighter — APPROVED** [review]

**Review by de5peratelighter — CHANGES_REQUESTED** [review]
Please resolve mentioned items

File: client/src/layouts/Footer.vue (hunk 2 above)
Comment (Danulo004) [reply client/src/layouts/Footer.vue]: Thanks, I am going to fix this!

**Comment by igorkokotko** [comment]
@Danulo004 conflicts

File: client/src/components/LikesList.vue [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+              <q-avatar color="primary" text-color="white">
+                <img :src="`https://cdn.quasar.dev/img/${like.avatar}`" />
+              </q-avatar>
+            </q-item-section>
+
+            <q-item-section @click="$router.push({ path: `/profile/${like.user_id}` })">
+              <q-item-label>{{ like.name }}</q-item-label>
+            </q-item-section>
+            <q-item-section side>
+              <q-item-label caption>{{ $moment( parseInt(like.date) ).fromNow() }}</q-item-label>
```
Comment (igorkokotko) [inline client/src/components/LikesList.vue:26]: Move to computed property

File: client/src/components/LikesList.vue [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+      position: (document.body.offsetWidth < 700) ? 'bottom' : 'right'
+    }
+  },
+  computed: {
+    ...mapGetters(['getLikes'])
+  },
+  watch: {
+    info: {
+      handler: (previous) => {
+        console.log(previous)
```
Comment (igorkokotko) [inline client/src/components/LikesList.vue:65]: Remove console

File: client/src/components/LikesList.vue [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+      },
+      deep: true
+    }
+  },
+  methods: {
+    ...mapActions(['getLikedList', 'updateList'])
+  },
+  created () {
+    console.log("Created")
```
Comment (igorkokotko) [inline client/src/components/LikesList.vue:74]: Console

File: client/src/router/index.js [hunk 7]
```diff
@@ -1,32 +1,61 @@
...
+{
+  path: '/settings',
+  name: 'Settings',
+  component: () => import('../views/Settings/index.vue'),
+  children: [{
+    path: 'editProfile',
+    component: () => import('../views/Settings/Tabs/EditProfile/index.vue')
   },
   {
+<<<<<<< HEAD
```
Comment (igorkokotko) [inline client/src/router/index.js:50]: WTF

**Review by igorkokotko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**